src/expr/src/relation/func.rs

```diff
@@ -23,6 +23,8 @@ use repr::{ColumnType, Datum, RelationType, Row, RowArena, ScalarType};
 
 use crate::scalar::func::jsonb_stringify;
 
+type Diff = isize;
```

Comment:
Having this type alias in two places seems wrong. If `dataflow_types::Diff` needs to be in `expr`, then it needs to be in `expr`—can you delete the one in `dataflow_types` and make all of its uses point to this one instead?